src/defs.h

```diff
@@ -308,6 +310,8 @@ struct insn {
     var_t *rs1;
     var_t *rs2;
     int sz;
+    bool useful;
```

Comment:
It would be better to define "useful."

Reply:
Do you mean to add comments to this section of the code to make it clearer?

Reply:
> Do you mean to add comments to this section of the code to make it clearer?

Exactly. Otherwise, it is hard to figure out what "useful" is.